### ISSUE TITLE
Still working? Specific Xcode / iOS I need to run to get it working?

### ISSUE THREAD

**Issue by stevenpaulhoward** [ninja342/Screen-Time#1]
Heyo long shot sending this out but I'm running Xcode 15 RC and trying to run this code to understand how the ScreenTime API works. It's been killing me trying to understand how to leverage the API to render a simple report, haven't been able to get it working and been testing a ton of different stuff. Your repo is the closest I've seen to anyone trying to use the DeviceActivity framework, and it seems like you got something working at one point to render a report of app time usage. Would love if you could share which Xcode version you were using so I can see if I can get this repo running on an older version. Any advice or tips on where to go to try and figure out how to get a device activity report rendered would also be immensely appreciated :) Thanks so much for sharing this repo!!!

**Comment by ninja342** [comment]
Hi @stevenpaulhoward I have tried to run the project with Xcode 14.3 and able to run it on simulator without a problem. But screen time not working properly on simulator. What error you’re getting while trying to run this project?

**Comment by stevenpaulhoward** [comment]
Hi @ninja342 thanks for the response! Sounds like you're seeing the same thing that I am seeing. Basically Family Controls authorization works, but the device activity doesn't register. It seems like maybe the device activity report or the device activity monitor isn't actually monitoring? I'm trying to debug this myself, will update here if I find anything that helps.

Here's a screen recording of what I'm seeing in the simulator in Xcode 15 RC

https://github.com/ninja342/Screen-Time/assets/40086690/147df4ca-1617-401f-accd-f486f0cc39ee

**Comment by ninja342** [comment]
Yes it works like this on a simulator. Also you can check from settings->screen time that iOS settings also does not updating screen time values on simulator. So to test this feature you need to install it on a real device because simulators are not working properly for this feature. Currently I am not able to install to my device because my Xcode is outdated but I am updating it and will try it on my device.

**Comment by stevenpaulhoward** [comment]
Wow, I had no clue! You're the first person I've heard this from. Thank you so much for sharing, here I was feeling stupid for the last 8+ days that I was just not able to make this thing work!  😅 Thank you, I will try to send it to my iPhone and see if that gets it to work!
